save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/VulnerabilityWelcomeView.kt

```diff
@@ -89,3 +102,29 @@ val vulnWelcomeView: FC<WelcomeProps> = FC { props ->
         }
     }
 }
+
+fun ChildrenBuilder.stats(vulnerabilities: List<VulnerabilityDto>) {
+    div {
+        className = ClassName("card mt-3")
+        div {
+            className = ClassName("mt-5 mb-5")
+            style = jso {
+                height = 15.rem
+            }
+            div {
+                className = ClassName("row justify-content-center")
+                ReactHTML.strong {
```

Comment:
we use html methods without `ReactHTML.*`, can you add imports for `ReactHtml.strong` and `ReactHtml.h1`?

Reply:
```
strong {
```